app/services/impl/ApiTogglService.scala

```diff
@@ -0,0 +1,33 @@
+package services.impl
+
+import services.TogglService
+import services.TogglService.Project
+import play.api.libs.ws.WSClient
+import scala.concurrent.{ExecutionContext, Future}
+import play.api.libs.ws.WSAuthScheme.BASIC
+import play.api.libs.json._
+
+/**
+  * Our application calls Toggl API, and as a result we receive Scala Future.
```

Comment:
This should be "Implementation of [[ TogglService]]" and the "result we recieve ..." doesn't make much sense here.